### DISCUSSION TITLE
Why not 3.2 branch

### DISCUSSION THREAD

**Discussion by eduardonunesp** [GodotECS/godex#219] [General]
Is it possible to patch to 3.2 branch as well?

**Comment by AndreaCatania** [comment]
Everything is self contained, doing a port should be easy. However, I don't have the time to also maintain the 3.2 version, but if someone is willing to do it, I'm all for it.

**Comment by eduardonunesp** [comment]
Just looking from the top, I see different conflicts and requirements that are tied to version 4, hope someone with good knowledge about the engine and time to backport this awesome job.

**Comment by AndreaCatania** [comment]
Converted to a discussion, so to keep the issues and non technical questions separated.
